staging/src/k8s.io/apiserver/pkg/apis/config/types.go

```diff
@@ -86,6 +86,9 @@ type IdentityConfiguration struct{}
 
 // KMSConfiguration contains the name, cache size and path to configuration file for a KMS based envelope transformer.
 type KMSConfiguration struct {
+	// apiVersion of KeyManagementService
+	// +optional
+	APIVersion string
```

Comment:
this field should be ensured to be populated in `validateKMSConfiguration`, and contain a supported value (currently v1 and v2)

Reply:
added in https://github.com/kubernetes/kubernetes/pull/111126/commits/6074a7a62c68c5e1d83297e257e089b02511ca51